build.assets/Dockerfile-centos7-fips

```diff
@@ -34,6 +33,10 @@ RUN yum install --nogpgcheck -y \
         ${DEVTOOLSET}-make && \
     yum clean all
 
+# Install custom package with -fPIC.
+COPY --from=teleport-buildbox-centos7-assets /opt/custom-packages /opt/custom-packages
+RUN rpm -ivh /opt/custom-packages/elfutils-libelf-devel-static.*.rpm
```

Comment:
The is one problem with this approach. RPM copied a line above will still take space in the final image even after removing them as they were copied in a different layer. How big are the RPMs?

Reply:
I'll remove `/opt/custom-packages` afterwards to solve this.